shared/utils/count_oval_objects.py

```diff
@@ -111,7 +111,19 @@ def main():
         rule_id = rule.attrib['id']
         oval_refs = []
         for ref in rule.findall(".//check-content-ref"):
-            oval_name = ref.attrib['name']
+
+            # Skip remotely referenced OVAL checks since they won't have the
+            # 'name' attribute set (just 'href' would be set in that case)
+            try:
+                oval_name = ref.attrib['name']
+            except KeyError:
+                if ref.attrib['href'] is not None:
```

Comment:
@mpreisler 
Looks you are correct. Here's is the output for ```check-content-ref``` without both ```name``` and ```href```:
```
$ grep -A4 '<fix system="urn:xccdf:fix:script:sh" id="security_patches_up_to_date">yum -y update' /tmp/test/scap-security-guide/RHEL/6/output/ssg-rhel6-xccdf.xml 
          <fix system="urn:xccdf:fix:script:sh" id="security_patches_up_to_date">yum -y update
</fix>
          <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5">
            <!-- <check-content-ref href="http://www.redhat.com/security/data/oval/Red_Hat_Enterprise_Linux_6.xml"/> -->
            <check-content-ref />

$ ./count_oval_objects.py /tmp/test/scap-security-guide/RHEL/6/output/ssg-rhel6-xccdf.xml 
kernel_disable_entropy_contribution_for_solid_state_drives: textfilecontent54_object
partition_for_tmp: partition_object
partition_for_var: partition_object
partition_for_var_log: partition_object
partition_for_var_log_audit: partition_object
partition_for_home: partition_object
encrypt_partitions:
ensure_redhat_gpgkey_installed: rpminfo_object
ensure_gpgcheck_globally_activated: textfilecontent54_object
ensure_gpgcheck_never_disabled: textfilecontent54_object
Traceback (most recent call last):
  File "./count_oval_objects.py", line 140, in <module>
    main()
  File "./count_oval_objects.py", line 120, in main
    if ref.attrib['href'] is not None:
KeyError: 'href'
```

Will change this yet.